lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1679,24 +1680,32 @@ def test_certificate_generation_all_whitelisted(self):
         }
         self.assertCertificatesGenerated(task_input, expected_results)
 
-    def test_certificate_generation_whitelist_already_generated(self):
+    @ddt.data(
+        CertificateStatuses.downloadable,
+        CertificateStatuses.generating,
+        # CertificateStatuses.regenerating, TODO: why is this failing?
```

Comment:
@adampalay according to my understanding, this case is failing because of `regenerating status` is not add in [valid_statuses](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/certificates/queue.py#L217). If we add `status.regenerating` here, this will fix it.

FYI: @Shrhawk